### PR TITLE
Add limb drawing example

### PR DESCRIPTION
**Pull request by Cadair**
See the mailing list for inspiration.

### CODE REVIEW

File: examples/gallery/plot_AIA_limb_STEREO.py [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+# early 2011 when the STEREO spacecraft were roughly 90 deg seperated from the
+# Earth.
+
+stereo = (vso.attrs.Source('STEREO_B') &
+          vso.attrs.Instrument('EUVI') &
+          vso.attrs.Time('2011-01-01', '2011-01-01T00:10:00'))
+
+aia = (vso.attrs.Instrument('AIA') &
+       vso.attrs.Sample(24 * u.hour) &
+       vso.attrs.Time('2011-01-01', '2011-01-02'))
```
Comment (dpshelio) [inline examples/gallery/plot_AIA_limb_STEREO.py]: Why not use the same time ranges for both images?  If I remember correctly the notebook had an image from a few minutes away... maybe if they are closer in time would be better?

File: examples/gallery/plot_AIA_limb_STEREO.py [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+maps = {m.detector: m.submap((-1100, 1100) * u.arcsec,
+                             (-1100, 1100) * u.arcsec) for m in sunpy.map.Map(files)}
+
+
+##############################################################################
+# Calculate points on the limb in the AIA image for the half that can be seen
+# from STEREO.
+
+r = maps['AIA'].rsun_obs - 1 * u.arcsec  # remove the one arcsec so it's on disk.
+# Adjust the following range if you only want to plot on STEREO_A
```
Comment (dpshelio) [inline examples/gallery/plot_AIA_limb_STEREO.py]: Isn't on Stereo B??

File: examples/gallery/plot_AIA_limb_STEREO.py (hunk 1 above)
Comment (Cadair) [inline examples/gallery/plot_AIA_limb_STEREO.py]: sample is only supported for the jsoc provider, so I have to restrict the time range for STEREO manually. These are the closest times available.

File: examples/gallery/plot_AIA_limb_STEREO.py (hunk 2 above)
Comment (Cadair) [inline examples/gallery/plot_AIA_limb_STEREO.py]: yes, so you would have to change these numbers if you wanted to plot on A instead.

Comment (dpshelio) [inline examples/gallery/plot_AIA_limb_STEREO.py]: I see :)

File: examples/gallery/plot_AIA_limb_STEREO.py (hunk 1 above)
Comment (dpshelio) [inline examples/gallery/plot_AIA_limb_STEREO.py]: What I meant to use a sdo image closer to the stereo image, instead the other way around

File: examples/gallery/plot_AIA_limb_STEREO.py (hunk 2 above)
Comment (nabobalis) [inline examples/gallery/plot_AIA_limb_STEREO.py]: How would you adjust the range if you only wanted to plot on stereo A?

Comment (Cadair) [inline examples/gallery/plot_AIA_limb_STEREO.py]: you would need it to be 0-180 deg, so it's the other side of the SDO limb.

**Comment by Cadair** [comment]
I added the overlay example from last night as well for good measure.

**Comment by dpshelio** [comment]
I reaprove the last example!! Thanks @Cadair !!!

**Comment by Cadair** [comment]
Summon the @bsipocz, Travis has broken.

**Comment by Cadair** [comment]
@bsipocz is this a known bug or something terrible we have done?

**Comment by bsipocz** [comment]
Nothing is broken as far as I know. Try using ``SPHINX_VERSION=1.5.6``, compared to your last passing build it's now seems to be upgraded to 1.6.1.

My second best guess is to compare the version number of the other packages that gets installed, using ``DEBUG=1`` provides a more verbose log.

**Comment by Cadair** [comment]
@bsipocz looks like the sphinx version might be the thing, I guess it may be related to this: https://github.com/sphinx-gallery/sphinx-gallery/issues/241

**Comment by bsipocz** [comment]
@Cadair - Good catch. I see that astropy travis still uses 1.5.6, I guess due to some caching issues, so we will probably run into this same issue when the new version gets picked up.

**Comment by Cadair** [comment]
Seems to have done the job, hopefully RTD will not suffer the same fate.

**Comment by bsipocz** [comment]
I'm thinking to put the workaround directly in ci-helpers, so it works for all packages that use ``sphinx-gallery``

**Pull request merged**